lib/omnibus/packagers/solaris.rb

```diff
@@ -58,6 +50,10 @@ def install_dirname
     def install_basename
       File.basename(project.install_dir)
     end
+    
+    def staging_dir_path(fileName)
+      File.join(staging_dir,fileName)
```

Comment:
A few small Ruby style things:

* Don't camel case variable names. `fileName` should be `file_name`.
* Add a space after the comma: `staging_dir, file_name`.